tests/ccl_test_cmblens.c

```diff
@@ -0,0 +1,91 @@
+#include "ccl.h"
+#include "../include/ccl_params.h"
+#include "ctest.h"
+#include <stdlib.h>
+#include <stdio.h>
+#include <math.h>
+#include <string.h>
+
+#define CLS_TOLERANCE 1E-3
+#define CLS_FRACTION 1E-3
+
+CTEST_DATA(cls) {
+  double Omega_c;
+  double Omega_b;
+  double h;
+  double A_s;
+  double n_s;
+  double sigma_8;
+};
+
+CTEST_SETUP(cls) {
+  data->Omega_c = 0.30;
+  data->Omega_b = 0.00;
+  data->h = 0.7;
+  data->A_s = 2.1e-9;
+  data->sigma_8=0.8;
+  data->n_s = 0.96;
+}
+
+static int linecount(FILE *f)
+{
+  //////
+  // Counts #lines from file
+  int i0=0;
+  char ch[1000];
+  while((fgets(ch,sizeof(ch),f))!=NULL) {
+    i0++;
+  }
+  return i0;
+}
+
+static void compare_cls(struct cls_data * data)
+{
+  int status=0;
+
+  ccl_configuration config = default_config;
+  config.transfer_function_method = ccl_bbks;
+  config.matter_power_spectrum_method = ccl_linear;
+  ccl_parameters params = ccl_parameters_create_flat_lcdm(data->Omega_c,data->Omega_b,data->h,
+							  data->A_s,data->n_s, &status);
+  params.Omega_k=0;
+  params.Omega_g=0;
+  params.Omega_n_rel=0;
+  params.Omega_l = 1.0 - params.Omega_m;
+  params.sigma_8=data->sigma_8;
+  ccl_cosmology * cosmo = ccl_cosmology_create(params, config);
+  ASSERT_NOT_NULL(cosmo);
+
+  //Fix spline parameters for the high-redshift needs of the CMB lensing power spectrum
+  ccl_splines->A_SPLINE_NA=10000;
+  ccl_splines->N_A=500;
+
+  char fname[256];
+  FILE *fi_cc;
+  CCL_ClTracer *tr_cl=ccl_cl_tracer_cmblens_new(cosmo,1090.,&status);
+  ASSERT_NOT_NULL(tr_cl);
+  sprintf(fname,"tests/benchmark/codecomp_step2_outputs/run_log_cl_cc.txt");
```

Comment:
Is this benchmark really coming from the code comparison step 2? I forgot we had compared CMB lensing outputs as well. Can you confirm?

Reply:
We didn't. I generated my own benchmark using the same code I used to generate the benchmarks from phase 2 of the code comparison. I mentioned this above - happy to take in any external benchmarks!